javascript/README.md

```diff
@@ -759,44 +759,71 @@ function sayHi(name) {
 
 ## Functions
 
-### 6.1 Function Style
+### 6.1 Avoid Function Hoisting
 
-Understand the difference between **function declarations** and **function expressions**.
+Although it is possible to call functions before they are defined via hoisting we prefer to avoid this pattern in our code as it can be confusing.
```

Comment:
@Paul-Hebert Can you link to https://developer.mozilla.org/en-US/docs/Glossary/Hoisting

Reply:
Sure, makes sense

Reply:
```suggestion
Although it is possible to call functions before they are defined via [hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) we prefer to avoid this pattern in our code as it can be confusing.
```

Like this?

Reply:
👍